shared/uavobjectdefinition/hwshared.xml

```diff
@@ -67,7 +67,7 @@
 		</field>
 		<field name="MagOrientation" units="function" type="enum" elements="1" options="Top0degCW,Top90degCW,Top180degCW,Top270degCW,Bottom0degCW,Bottom90degCW,Bottom180degCW,Bottom270degCW" defaultvalue="Top0degCW"/>
 		<field name="USB_HIDPort" units="function" type="enum" elements="1" options="USBTelemetry,Disabled" defaultvalue="USBTelemetry"/>
-		<field name="USB_VCPPort" units="function" type="enum" elements="1" options="USBTelemetry,ComBridge,DebugConsole,Disabled" defaultvalue="Disabled"/>
+		<field name="USB_VCPPort" units="function" type="enum" elements="1" options="USBTelemetry,ComBridge,DebugConsole,MSP,Disabled" defaultvalue="MSP"/>
```

Comment:
Right now all targets override it.

It's kinda neat when default is MSP, because of the fallback-to-telemetry behavior.  It allows you to use VCP telemetry without conflicting over USB telem ids / that legacy infrastructure.